rockset/provider.go

```diff
@@ -155,6 +155,17 @@ func toStringPtrNilIfEmpty(v string) *string {
 	return &v
 }
 
+func toBoolPtrNilIfEmpty(v interface{}) *bool {
+	var res *bool
+	if v == nil {
+		return res
+	} else {
+		vB := v.(bool)
```

Comment:
Just a FYI, this is dangerous as if e.g. an `int` is passed in, this will panic. It can be avoided using
```
if vB, ok := v.(bool); !ok {
  return errors.New("not a bool")
}
```
but it requires changing the function signature

Reply:
gotcha, i can make the update

Reply:
is the type not enforced by terraform here? Or can i not rely on that

Reply:
You're right, the schema definition makes sure it'll be a `bool`, but if we change the schema type there is no way to detect that change at compile time, so we need to rely on having test coverage.

Reply:
gotcha, I'll add testing